src/test/java/jp/co/yahoo/yosegi/message/formatter/json/TestJacksonObjectFormatter.java

```diff
@@ -0,0 +1,363 @@
+      }
+      return null;
+    }
+
+    @Override
+    public PrimitiveObject get( final int index ) throws IOException {
+      if ( index == 0 ) {
+        return new StringObj( "a" );
+      } else if ( index == 1 ) {
+        return new StringObj( "b" );
+      } else if ( index == 2 ) {
+        return new StringObj( "c" );
+      }
+      return null;
+    }
+
+    @Override
+    public IParser getParser( final String key ) throws IOException {
+      return new JacksonNullParser();
+    }
+
+    @Override
+    public IParser getParser( final int index ) throws IOException {
+      return new JacksonNullParser();
+    }
+
+    @Override
+    public String[] getAllKey() throws IOException {
+      return new String[]{ "key1" , "key2" , "key3" };
+    }
+
+    @Override
+    public boolean containsKey( final String key ) throws IOException {
+      if ( "key1".equals( key ) ) {
+        return true;
+      } else if ( "key2".equals( key ) ) {
+        return true;
+      } else if ( "key3".equals( key ) ) {
+        return true;
+      }
+      return false;
+    }
+
+    @Override
+    public int size() throws IOException {
+      return 3;
+    }
+
+    @Override
+    public boolean isArray() throws IOException {
+      return isArray;
+    }
+
+    @Override
+    public boolean isMap() throws IOException {
+      return ! isArray;
+    }
+
+    @Override
+    public boolean isStruct() throws IOException {
+      return ! isArray;
+    }
+
+    @Override
+    public boolean hasParser( final int index ) throws IOException {
+      return false;
+    }
+
+    @Override
+    public boolean hasParser( final String key ) throws IOException {
+      return false;
+    }
+
+    @Override
+    public Object toJavaObject() throws IOException {
+      return null;
+    }
+
+  }
+
+  @Test
+  public void T_write_1() throws IOException {
+    Map<Object,Object> map = new HashMap<Object,Object>();
+    map.put( "key1" , "a" );
+    map.put( "key2" , "b" );
+    map.put( "key3" , "c" );
+    JacksonObjectFormatter formatter = new JacksonObjectFormatter();
+    JsonNode node = formatter.write( map );
+    assertTrue( ( node instanceof ObjectNode ) );
+
+    ObjectNode objNode = (ObjectNode)node;
+    assertEquals( objNode.size() , 3 );
+    assertTrue( objNode.get("key1").isTextual() );
+    assertTrue( objNode.get("key2").isTextual() );
+    assertTrue( objNode.get("key3").isTextual() );
+    assertEquals( objNode.get("key1").asText() , "a" );
+    assertEquals( objNode.get("key2").asText() , "b" );
+    assertEquals( objNode.get("key3").asText() , "c" );
+  }
+
+  @Test
+  public void T_write_3() throws IOException {
+    JacksonObjectFormatter formatter = new JacksonObjectFormatter();
+    JsonNode node = formatter.write( null );
+    assertTrue( ( node instanceof ObjectNode ) );
+
+    ObjectNode objNode = (ObjectNode)node;
+    assertEquals( objNode.size() , 0 );
+  }
+
+  @Test
+  public void T_write_4() throws IOException {
```

Comment:
Please write the test method name is to describe what is wanted to in the test instead of using number (it makes vague one).

Reply:
I changed the method name.